src/System.Management/src/System/Management/ManagementDateTime.cs

```diff
@@ -119,111 +119,93 @@ public static DateTime ToDateTime(string dmtfDate)
                 }
                 tempString = dmtf.Substring(4, 2);
                 if (("**" != tempString)) 
                 {
-                    month = System.Int32.Parse(tempString,frmInt32);
+                    month = int.Parse(tempString,frmInt32);
                 }
                 tempString = dmtf.Substring(6, 2);
                 if (("**" != tempString)) 
                 {
-                    day = System.Int32.Parse(tempString,frmInt32);
+                    day = int.Parse(tempString,frmInt32);
                 }
                 tempString = dmtf.Substring(8, 2);
                 if (("**" != tempString)) 
                 {
-                    hour = System.Int32.Parse(tempString,frmInt32);
+                    hour = int.Parse(tempString,frmInt32);
                 }
                 tempString = dmtf.Substring(10, 2);
                 if (("**" != tempString)) 
                 {
-                    minute = System.Int32.Parse(tempString,frmInt32);
+                    minute = int.Parse(tempString,frmInt32);
                 }
                 tempString = dmtf.Substring(12, 2);
                 if (("**" != tempString)) 
                 {
-                    second = System.Int32.Parse(tempString,frmInt32);
+                    second = int.Parse(tempString,frmInt32);
                 }
                 tempString = dmtf.Substring(15, 6);
                 if (("******" != tempString)) 
                 {
-                    ticks = (System.Int64.Parse(tempString,(IFormatProvider)CultureInfo.InvariantCulture.GetFormat(typeof(System.Int64)))) * (System.TimeSpan.TicksPerMillisecond/1000);
+                    ticks = (long.Parse(tempString,(IFormatProvider)CultureInfo.InvariantCulture.GetFormat(typeof(Int64)))) * (TimeSpan.TicksPerMillisecond/1000);
                 }
-                if( year < 0 || month < 0 || day < 0 || hour < 0 || minute < 0 || second < 0 || ticks < 0)
+                tempString = dmtf.Substring(22, 3);
+                if (("***" != tempString)) 
                 {
-                    throw new System.ArgumentOutOfRangeException("dmtfDate");
+                    tempString = dmtf.Substring(21, 4);
+                    utcOffset = int.Parse(tempString,frmInt32);
                 }
 
+                if( year < 0 || month < 0 || day < 0 || hour < 0 || minute < 0 || second < 0 || ticks < 0)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(dmtfDate));
+                }
             }
             catch
             {
-                throw new System.ArgumentOutOfRangeException("dmtfDate");
+                throw new ArgumentOutOfRangeException(nameof(dmtfDate));
             }
 
 
             // Construct a new System.DateTime object
-            datetime = new System.DateTime(year, month, day, hour, minute, second, millisec);
+            var datetime = new DateTime(year, month, day, hour, minute, second, millisec);
```

Comment:
>  var datetime = new DateTime(year, month, day, hour, minute, second, millisec); [](start = 10, length = 80)

nit: It would be nice to explicitly specify DateTimeKind.UTC here to help in code readability and debugging too.